### ISSUE TITLE
Fix test suite

### ISSUE THREAD

**Issue by damcclean** [doublethreedigital/cookie-notice#23]
Everytime I try to run the test suite for this Statamic addon/Laravel package, I seem to get errors pretty similar to:

```
Script ./vendor/bin/phpunit handling the test event returned with error code 2
Illuminate\Contracts\Container\BindingResolutionException: Target class [config] does not exist.
```

You can see an example of the tests [failing in CI over here](https://github.com/doublethreedigital/cookie-notice/runs/1188999896?check_suite_focus=true).

These tests need to be fixed at some point so I can have confidence when building new features or merging bug fix pull requests from contributors.

**Comment by bitasterisk** [comment]
I would like to work on this issue.

**Comment by bitasterisk** [comment]
i am doing.

**Comment by damcclean** [comment]
Closed by #24
